README.md

```diff
@@ -19,26 +19,38 @@ module "servicebus-subscription" {
 }
 ```
 
-## Variables
+## azurerm_servicebus_subscription_rule
 
-### Configuration
-
-The following parameters are required by this module
-
-- `name` the name of the ServiceBus namespace.
-- `resource_group_name` the name of the resource group in which to create the ServiceBus namespace.
-- `namespace_name` the name of the service bus namespace in which the subscription must be created.
-- `topic_name` the name of the service bus topic in which the subscription must be created.
+The following example shows how to give read access to a user assigned managed identity for the subscription:
```

Comment:
I can't see this? looks like a copy paste error from line 42

Reply:
updated it now